web-marketplace/src/hooks/projects/useProjectsWithOrders.ts

```diff
@@ -88,7 +88,7 @@ export function useProjectsWithOrders({
       }),
     );
 
-  const { data: sellOrders, isFetching: isLoadingSellOrders } = useQuery(
+  const { data: sellOrders, isLoading: isLoadingSellOrders } = useQuery(
```

Comment:
we refetch sell orders before opening the buy modal, which was causing isFetching here to be true again, using isLoading instead to take only the initial load into account.